src/main/java/org/openrewrite/java/migrate/lombok/LombokValueToRecord.java

```diff
@@ -42,7 +40,7 @@
 @EqualsAndHashCode(callSuper = false)
 public class LombokValueToRecord extends ScanningRecipe<Map<String, Set<String>>> {
 
-    private static final AnnotationMatcher LOMBOK_VALUE_MATCHER = new AnnotationMatcher("@lombok.Value");
+    private static final AnnotationMatcher LOMBOK_VALUE_MATCHER = new AnnotationMatcher("@lombok.Value()");
```

Comment:
This was an attempt to have the annotation matcher only match annotations without arguments; that didn't work, but I think it should.